pkg/cmd/cli/cmd/importimage.go

```diff
@@ -249,151 +249,134 @@ func (o *ImportImageOptions) waitForImport(resourceVersion string) (*imageapi.Im
 }
 
 func (o *ImportImageOptions) createImageImport() (*imageapi.ImageStream, *imageapi.ImageStreamImport, error) {
+	var isi *imageapi.ImageStreamImport
 	stream, err := o.isClient.Get(o.Name)
-	from := o.From
-	tag := o.Tag
-
+	// no stream, try creating one
 	if err != nil {
 		if !errors.IsNotFound(err) {
 			return nil, nil, err
 		}
 		if !o.Confirm {
```

Comment:
Not quite sure, I'd prefer to leave it as is. I feel more comfortable with warning users than forcing them to delete afterwards. They'll end up wondering what happened and that's not desirable imho.